clients/src/main/java/org/apache/kafka/clients/FetchSessionHandler.java

```diff
@@ -454,7 +454,7 @@ String verifyFullFetchResponsePartitions(Set<TopicPartition> topicPartitions, Se
      * @param topicPartitions  The topicPartitions from the FetchResponse.
      * @param ids              The topic IDs from the FetchResponse.
      * @param version          The version of the FetchResponse.
-     * @return                 True if the incremental fetch response partitions are valid.
+     * @return                 null if the incremental fetch response partitions are valid; human-readable problem description otherwise.
```

Comment:
Thanks for the nice catch! I'm wondering if the L467 and L469 are wrong. Currently, we did:
```java
       if (extra.isEmpty())
            bld.append("extraPartitions=(").append(Utils.join(extra, ", ")).append("), ");
        if (extraIds.isEmpty())
            bld.append("extraIds=(").append(Utils.join(extraIds, ", ")).append("), ");
```
And that will only append **empty** extras and extraIds, which is not right. Do you think it should be non-empty check?

Reply:
These should be non-empty checks. Thanks for catching.